src/details/ArborX_DetailsFDBSCANDenseBox.hpp

```diff
@@ -274,14 +274,17 @@ void unionFindWithinEachDenseCell(ExecutionSpace const &exec_space,
                                   CellIndices sorted_dense_cell_indices,
                                   Permutation permute, UnionFind union_find)
 {
+  auto const n = sorted_dense_cell_indices.size();
+  if (n == 0)
```

Comment:
```suggestion
  if (n <= 1)
```
otherwise there is still an empty range.

Reply:
Sure, thanks.